platforms/ide/ide-native/build.gradle.kts

```diff
@@ -1,5 +1,7 @@
 plugins {
-    id("gradlebuild.distribution.api-java")
+    // Uninstrumented since it is a mix of Groovy and Java code
+    // and we don't plan to have upgrades for IDE plugins
+    id("gradlebuild.distribution.uninstrumented.api-java")
```

Comment:
💭 This is uninstrumented since we have some Groovy classes. Alternatively we can also disable ann. processor via compiler arguments